### ISSUE TITLE
Run-time error '52': Bad file name or number

### ISSUE THREAD

**Issue by agnesnash** [sotirop/mergelook#5]
Upon executing the macro "Send emails" button, the following error is displayed:
Run-time error '52':
Bad file name or number

Debugger highlights the following row in the code:
FileExists = (Len(Dir(path_)) > 0)

No files were modified, tested with the downloaded package as is. I've been using this tool for years now with no issues and it helps SO MUCH in my day to day work. But now suddenly it doesn't work anymore :(

Can you please help?

**Comment by sotirop** [comment]
Please post the files that produce the error.

**Comment by agnesnash** [comment]
Those were the original files, downloaded right from here without any
modifications. However, I already found the issue - it was due to the long
path of the file due to OneDrive. Our company recently synced all of our
files to OneDrive, so the path while on my desktop was something like
"Companyname - Company OneDrive/username/Desktop" something. As soon as I
moved the whole tool folder to a shared drive with a short path, the tool
worked just fine.
Thank you!
